### ISSUE TITLE
Clear the contents of the People Picker control

### ISSUE THREAD

**Issue by SuleymanA** [pnp/sp-dev-fx-controls-react#1569]
#### Category
[ X] Question

#### Version
3.7.0 

I have seen other questions about this topic but cannot find any clear answer. Our usecase is the following:
- User selects a user with the picker
- OnChange event we validate the user (business rules)
- If a invalid user we want to remove the user and show error message

We are looking for a way to clear the currently selected user from the People picker control. We tried with the defaultSelectedUsers property but after initial render it doesn't have any effect. 

Could someone please advise how the clear people picker selection?

NOTE: I know we are not using the latest version but I don't see any new properties that are added to achive this. I was also not able to find any relevant fixes/changes in the release notes.

**Comment by GuidoZam** [comment]
Hi @SuleymanA, I think that you can achieve this by calling the onChange method with an empty array.
I tried getting a reference of the PeoplePicker control and inside the onChange I did this:

`onChange={(items: any[]) => { if (items.length > 0) { peoplePickerRef.onChange([]); }}}`

and the value got cleared, pay attention to loop calls!

**Comment by SuleymanA** [comment]
Thx for your help @GuidoZam . I tried with calling the onChange from the reference object. I could only call it via "ref.props.onChange". This only triggered my custom onChange event receiver. The selection in the picker didn't change. I did manage to get it cleared by directly updating the state property of the component called "selectedPersons". Control clears the selection when putting a empty array in the state property. 

But this workaround doesnt feel good. I think the control misses functionality to actually clear its selection. Would be nice if someone from the developer community can have a look.
